wodles/docker-listener/DockerListener.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+
+
+try:
+    import docker
+except:
+    print("'docker' module needs to be installed. Execute 'pip install docker' to do it.")
+    exit(1)
+import threading
+import json
+import socket
+import sys
+import time
+
+
+class DockerListener:
+
+    wait_time = 5
+    field_debug_name = "Wodle event"
+
+    def __init__(self):
+        """"
+        DockerListener constructor
+
+        """
+        # socket variables
+        if sys.platform == "win32":
+            self.wazuh_path = 'C:\Program Files (x86)\ossec-agent'
```

Comment:
I think this is not necessary since the wodle is going to exit without using this variable